app/templates/components/round-resource.hbs

```diff
@@ -1,4 +1,22 @@
 {{#link-to 'round' round tagName='div'}}
+  {{#paper-card as |card|}}
+    {{#card.title as |title|}}
+      {{#title.text as |text|}}
+        {{#text.headline}}{{round.course.name}}{{/text.headline}}
+        {{#text.subhead}}{{moment-format round.createdAt 'MMMM DD, YYYY'}}{{/text.subhead}}
+      {{/title.text}}
+      {{#title.media size="md"}}
+        <div class="score par-status belowPar">-4</div>
```

Comment:
Hmm... why should we show just the "logged in user's" round score? Maybe it'd be nice to see everyone on the card's score? Or is that information overload?